cmd/traffic_layout/traffic_layout.cc

```diff
@@ -50,7 +50,8 @@ const ArgumentDescription argument_descriptions[] = {
   {"init", 'i', "Initialize the ts_runroot sandbox (details in traffic_layout --init -h)", nullptr, nullptr, nullptr, nullptr},
   {"remove", 'r', "remove the ts_runroot sandbox (details in traffic_layout --remove -h)", nullptr, nullptr, nullptr, nullptr},
   {"force", '-', "force flag for init (details in traffic_layout --force -h)", nullptr, nullptr, nullptr, nullptr},
-
+  {"relocate", '-', "relocate the sandbox directory in a good form (details in traffic_layout --relocate -h)", nullptr, nullptr,
```

Comment:
You don't need to add "details in ...." to the usage message. Long form exposition can be added to the man page.

Reply:
Some simple usage examples are good to use. When we type in -h for a specific command like --init or --remove, a corresponding usage example will show up. People usually don't want to look into man page for such usage.